### DISCUSSION TITLE
gcode for creality cr-10s pro v2

### DISCUSSION THREAD

**Discussion by homerfoster64** [jneilliii/OctoPrint-BedLevelVisualizer#572] [General]
hi

i cannot seem to get this to work - tried

G28
G29 T

nothing happened

tried the example bilinear and it rose on the z access until it reach the top and tired to keep going up!

any suggestions

thanks

**Comment by jneilliii** [comment]
My suggestion is to figure out the commands that you need to run to probe your bed. In it's simplest form the plugin should work with 

```
G28
@BEDLEVELVISUALIZER
G29
```
for bilinear auto bed leveling configured in firmware.

**Reply by homerfoster64** [reply]
ok will give that a try and let you know when current job finishes

thanks for the quick reply

**Reply by homerfoster64** [reply]
loaded and now works great - thanks
